target/linux/realtek/dts/rtl8380_netgear_gs108t-v3.dts

```diff
@@ -0,0 +1,202 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+// Memory
+// ------
+// Spec: 256 MB DDR3-1600
```

Comment:
Are you sure about this? I recently discovered my mistake for the GS110TPP, which only has 128MB or RAM.

As @bkobl noted, this info should (also) go into the commit message. Since some Realtek switches have one MAC-address per port, and others (like this one) have one MAC-address for the entire device, this should probably also be mentioned in the commit message.

Reply:
Commented below, but adding more context here.  The datasheet for the chip indicates that it contains 256MB of memory.  However, U-Boot reports 128MB of memory at boot time.  I have a suspicion that something in the SoC artificially limits the exposed memory to upsell "pro" versions of the same board.  This may require reverse engineering the U-Boot binary code, as the board support package in the GPL source dump is missing.